src/components/App/App.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from '../Header/Header';
+import FiltersBar from "../FilterBar/FiltersBar";
+import RestaurantsList from "../RestaurantsList/RestaurantsList";
+import {connect} from "react-redux";
+
+import {fetchRestaurants} from "../../actions/restaurantsActions";
+
+
+class App extends Component {
+
+    constructor(props){
+        super(props);
+        this.addQueryParam = this.addQueryParam.bind(this);
```

Comment:
With the right configuration, you can avoid writing this, and instead define the function itself as:
```jsx
class App extends Component {
  // ...
  addQueryParam = (paramToAdd, valueToAdd) => {
    ...
  };
}
```

And automatically get the benefit of the `bind`.